grpcclient/rnn_serving_client.go

```diff
@@ -0,0 +1,70 @@
+package grpcclient
+
+import (
+	"context"
+	"math/rand"
+	"strconv"
+	"strings"
+
+	pb "github.com/vhive-serverless/vSwarm-proto/proto/rnn_serving"
+)
+
+type RNNServingGenerator struct {
+	GeneratorBase
+}
+
+func (g *RNNServingGenerator) Next() Input {
+
+	countries := []string{
+		"French", "Czech", "Dutch", "Polish", "Scottish", "Chinese",
+		"English", "Italian", "Portuguese", "Japanese", "German",
+		"Russian", "Korean", "Arabic", "Greek", "Vietnamese", "Spanish", "Irish",
+	}
+
+	var pkt = g.defaultInput
+
+	var language string
```

Comment:
Can you make the random selection configurable like in the aes client please.

Make a switch case for unique, random and if you want linear but this is not needed.

For unique take always the same language and number of samples.

Random is great but it should be configurable.